diff --git a/node_modules/react-native-share/ios/RNShare.h b/node_modules/react-native-share/ios/RNShare.h
index 3ec51db..aec3520 100644
--- a/node_modules/react-native-share/ios/RNShare.h
+++ b/node_modules/react-native-share/ios/RNShare.h
@@ -9,7 +9,7 @@
 #endif
 
 #if RCT_NEW_ARCH_ENABLED
-#import <RNShareSpec/RNShareSpec.h>
+#import "RNShareSpec.h"
 #endif
 
 @interface RNShare : NSObject <RCTBridgeModule, UIDocumentPickerDelegate>
diff --git a/node_modules/react-native-share/ios/RNShare.mm b/node_modules/react-native-share/ios/RNShare.mm
index 7013e79..225c632 100644
--- a/node_modules/react-native-share/ios/RNShare.mm
+++ b/node_modules/react-native-share/ios/RNShare.mm
@@ -25,7 +25,7 @@
 #import "RNShareUtils.h"
 
 #if RCT_NEW_ARCH_ENABLED
-#import <RNShareSpec/RNShareSpec.h>
+#import "RNShareSpec.h"
 #endif
 
 @implementation RNShare
